test/rbenv.bats

```diff
@@ -149,39 +149,63 @@ OUT
 
 @test "not enough arguments for rbenv-install" {
   stub_ruby_build
+  stub rbenv-help 'install : echo "Usage: rbenv install"'
```

Comment:
Ah, I assumed the purpose of the test was to ensure rbenv-help gets called, and that rbenv-help's own tests ensure that it does the right thing.

It sounds like this test is actually testing two things: 
1. that rbenv-help is invoked correctly and at the correct times (too few args, too many args, -h flag)
2. that ruby-build contains the correct 'usage' preface comments.

Rather than merge the two assertions into the same tests, I think I'd rather create an explicit test for 2. It would be more intention-revealing than hiding the second assertion in the body of a stub. Since we can still rely on rbenv-help to do its thing correctly, I'll create two new tests that simply assert that `sed -ne '/^#/!q;s/.//;s/.//;1,4d;p' < rbenv-{install | uninstall}` contain the correct messaging. Cool?